src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/DeleteSavesModal.tsx

```diff
@@ -10,8 +11,25 @@ interface Props {
   onClose: () => void
 }
 
+const deleteCollectionUpdater = (store, collectionID) => {
```

Comment:
**suggestion**: Add TS for `store` and `collectionID`)

Reply:
on it